DoLahTest/ParserUnitTest.cpp

```diff
@@ -379,6 +387,130 @@ namespace DoLahTest {
         }
     }
 
+    // From here, integrated test on special cases of 'toDateFormat' for relative time descriptions.
+    // The time will auto-filled, and the behaviour changes depending on the time of running.
+    // Test will be disabled in a certain condition where it cannot run properly.
+    TEST_METHOD(DateGivenDayAndMonth) {
+        if (month == 1 && day == 1) {
+            Assert::IsTrue(true);
```

Comment:
It will be difficult. I thought about sharing the 'current time' in the whole system by initializing it at one point and just passing it around, but it will cause extra behaviour to be handled if the user is using the program around 0AM when the day is changing.